models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -185,7 +185,7 @@ public function getDimensions()
     }
 
     /**
-     * @return Asset
+     * @return Image
```

Comment:
The Trait is also be used for Document and Video

Reply:
will look at this at a later time